src/main/scala/com/abdulradi/opentracing/xray/v1/Format.scala

```diff
@@ -90,12 +135,22 @@ object Format {
       "version" -> obj.version.asJson
     )
   )
-  implicit val subSegmentFieldsEncoder: ObjectEncoder[SubsegmentFields] = dummy // TODO
+  implicit val subSegmentFieldsEncoder: ObjectEncoder[SubsegmentFields] = ObjectEncoder.instance[SubsegmentFields](obj =>
+    merge()(
```

Comment:
Any reason you are using `merge` instead of `JsonObject`?

Reply:
I have no idea :fearful: , One minute I'll fix it.